rootfs-builder/alpine/Dockerfile.in

```diff
@@ -10,9 +10,31 @@ RUN apk update && apk add \
     make \
     bash \
     gcc \
+    g++ \
+	wget \
+	tar \
+    cmake \
+	autoconf \
+	automake \
+	m4 \
+	coreutils \
+	binutils \
+	vim \
     musl-dev \
+	musl \
+	libc-dev \
     linux-headers \
     apk-tools-static \
     libseccomp \
     libseccomp-dev \
     curl
+
+# alpine doesn't support x86_64-unknown-linux-gnu
+# It only support x86_64-unknown-linux-musl. Even worse,
+# it doesn't support proc-macro, which is needed for serde_derive
+# -- FIXME
```

Comment:
If would be useful to capture this problem in a GitHub issue and reference the URL in this comment.